### PR TITLE
only company profiles can create property

### PR DESCRIPTION
**Pull request by JRB958**
Small PR concerning what can be seen from the dashboard based on the user type (PUBLIC or COMPANY)

before: public users were able to create properties because they had access to the "add property" button 

after:
- implement dashboard view based on the type of user (public or company)
- the option to create the property only shows for "COMPANY" roles now
Relates: #4

### CODE REVIEW

**Review by patrickmac3 — APPROVED** [review]
looks good to me. The 'actions' are different for the different profiles for users with different roles

**Review by NicholasWahome — CHANGES_REQUESTED** [review]
Please provide more documentation! Then the request will be approved. Thank you!

File: frontend/src/components/dashboard/DashBoard.js [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
 
 const DashBoard = () => {
+    const [role, setRole] = useState(null)
     let navigate = useNavigate();
 
     function handleGoToProperty() {
         navigate('/create-property');
     }
+
+    useEffect(() => {
```
Comment (NicholasWahome) [inline frontend/src/components/dashboard/DashBoard.js:20]: Add documentation here

File: frontend/src/components/dashboard/DashBoard.js [hunk 2]
```diff
@@ -1,27 +1,49 @@
...
+        let role = ''
```
Comment (NicholasWahome) [inline frontend/src/components/dashboard/DashBoard.js:21]: Add documentation here as well

**Review by NicholasWahome — CHANGES_REQUESTED** [review]
Files still not changed

**Review by NicholasWahome — APPROVED** [review]
From code management perspective: 
Comments have been added

I edited your comment @JRB958  to add a relation to the user story. Please don't forget that next time. But the requirements have now been met. You can merge!

**Pull request merged**